GOVERNANCE.md

```diff
@@ -0,0 +1,98 @@
+# Project Contribution Guidelines
+
+## Overview
+
+This is a meritocratic, consensus-based community project. Anyone with an
+interest in the project can join the community, contribute to the project design
+and participate in the decision making process. This document describes how that
+participation takes place and how to set about earning merit within the project
+community.
+
+## Roles And Responsibilities
+
+### Contributors
+
+Contributors are community members who contribute in concrete ways to the
+project. Anyone can become a contributor. There is no expectation of commitment
+to the project, no specific skill requirements and no selection process.
+
+In addition to their actions as users, contributors may also find themselves
+doing one or more of the following:
+
+* supporting new users (existing users are often the best people to support new
+  users)
+* reporting bugs
+* identifying requirements
+* providing graphics and web design
+* programming
+* assisting with project infrastructure
+* writing documentation
+* fixing bugs
+* adding features
+
+Contributors engage with the project through the issue tracker, writing or
+editing documentation, and participating in the community Slack channels. They
```

Comment:
To be consistent with section "Contribution Process"
```suggestion
editing documentation, and participating in the Oasis Community Slack channels. They
```
Also, convert `Oasis Community Slack channels` to a link.